admin/installer/setup_swarm_manager.sh

```diff
@@ -11,10 +11,10 @@ DOCKER_CERT_HOME='/root/.docker'
 # Create Swarm cluster, and publish Swarm cluster ID to S3 bucket.
 rm -rf /tmp/swarm-config
 mkdir -p /tmp/swarm-config
-docker pull swarm:1.0.1
-docker run --rm -v ${DOCKER_CERT_HOME}:${DOCKER_CERT_HOME} swarm create > /tmp/swarm-config/swarm_cluster_id
+docker pull swarm:1.2.1
```

Comment:
This step isn't strictly necessary since we already `docker pull swarm 1.2.1` in the image building scripts.